File: app/src/main/java/de/karzek/diettracker/presentation/main/diary/meal/viewStub/CaloryDetailsView.java
package de.karzek.diettracker.presentation.main.diary.meal.viewStub;

import android.view.View;
import android.widget.TextView;

import com.mikhaellopez.circularprogressbar.CircularProgressBar;

import butterknife.BindView;
import butterknife.ButterKnife;
import de.karzek.diettracker.R;
import lombok.Data;

/**
 * Created by MarjanaKarzek on 31.05.2018.
 *
 * @author Marjana Karzek
 * @version 1.0
 * @date 31.05.2018
 */
@Data
public class CaloryDetailsView {

    @BindView(R.id.circle_progress_bar_calories)
    CircularProgressBar caloryProgressBar;
    @BindView(R.id.circle_progress_bar_calories_value)
    TextView caloryProgressBarValue;
    @BindView(R.id.circle_progress_bar_calories_max_value)
    TextView caloryProgressBarMaxValue;

    public CaloryDetailsView(View view) {
        ButterKnife.bind(this, view);
    }

}

File: app/src/main/java/de/karzek/diettracker/presentation/main/diary/meal/viewStub/CaloryDetailsView.kt
package de.karzek.diettracker.presentation.main.diary.meal.viewStub

import android.view.View
import android.widget.TextView

import com.mikhaellopez.circularprogressbar.CircularProgressBar

import de.karzek.diettracker.R


/**
 * Created by MarjanaKarzek on 31.05.2018.
 *
 * @author Marjana Karzek
 * @version 1.0
 * @date 31.05.2018
 */
open class CaloryDetailsView(view: View) {

    var caloryProgressBar: CircularProgressBar = view.findViewById(R.id.circle_progress_bar_calories)
    var caloryProgressBarValue: TextView = view.findViewById(R.id.circle_progress_bar_calories_value)
    var caloryProgressBarMaxValue: TextView = view.findViewById(R.id.circle_progress_bar_calories_max_value)

}
